### ISSUE TITLE
Redux Middleware

### ISSUE THREAD

**Issue by hydrotik** [keokilee/react-typescript-boilerplate#50]
Have you tried adding any redux middleware like thunk or logger?

**Comment by keokilee** [comment]
In our Node Knockout project, we did add redux-thunk in and it worked. I can probably add it to the boilerplate. Just need to make sure everything passes the linter (which was really hard to please).

**Comment by hydrotik** [comment]
Yeah, I tried importing it in with tsd and I am getting errors, had to pull the code out into a utils folder and pull it in that way, If you manage to get it in there, I'd love to take a look.
